app/models/card.rb

```diff
@@ -11,9 +11,21 @@ class Card < ActiveRecord::Base
   validates :original_text, :translated_text, presence: true
   validates_with CompareValidator
 
-  private
+  scope :random_card, -> { where('review_date <= ?', Time.current).order("RANDOM()").take }
```

Comment:
Line is too long. [91/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.